sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/KalixMethod.scala

```diff
@@ -95,9 +95,9 @@ case class VirtualServiceMethod(component: Class[_], methodName: String, inputTy
  * Build from methods annotated with @Subscription. Those methods are not annotated with Spring REST annotations, but
  * they become a REST method at the end.
  */
-case class RestServiceMethod(javaMethod: Method) extends AnyServiceMethod {
+case class RestServiceMethod(javaMethod: Method, inputTypeParamIndex: Int = 0) extends AnyServiceMethod {
```

Comment:
ahah!

Reply:
what was that? :D

Reply:
I think I added one `h` to much

It was intended to be: Aha! 

Like in: Aha! Now we have a solution for skipping the first arg and extracting the next one.

Reply:
Aha! Now I get it! :D